crates/bitwarden-vault/src/cipher/attachment.rs

```diff
@@ -36,6 +36,18 @@ pub struct AttachmentView {
     pub size_name: Option<String>,
     pub file_name: Option<String>,
     pub key: Option<EncString>,
+    /// The decrypted attachmentkey in base64 format.
+    ///
+    /// **TEMPORARY FIELD**: This field is a temporary workaround to provide
+    /// decrypted attachment keys to the TypeScript client during the migration
+    /// process. It will be removed once the encryption/decryption logic is
+    /// fully migrated to the SDK.
+    ///
+    /// **Ticket**: <https://bitwarden.atlassian.net/browse/PM-23005>
+    ///
+    /// Do not rely on this field for long-term use.
+    #[cfg(feature = "wasm")]
```

Comment:
Question: As far as I'm aware, we don't use the SDK to make API calls on mobile yet. I think this means that the same bug *has* to be present on mobile currently. We may need to expose this to mobile too?

Reply:
You are probably thinking about this [PR](https://github.com/bitwarden/server/pull/5998) which addresses the current issue we have in production

Reply:
No, this is the correct PR. I guess I'm wondering: Why do we need to expose the key material on WASM but don't for uniffi / mobile. Mobile also supports editing ciphers that have cipherkeys and attachments, and would downgrade them right?

Is the difference that we don't the sdk for *encryption* of attachment views / cipher views yet, wheras on mobile we do?